### ISSUE TITLE
participate to techempower

### ISSUE THREAD

**Issue by 2vg** [2vg/mofuw#29]
participate to techempower's benchmark

[bad](https://github.com/TechEmpower/FrameworkBenchmarks/pull/3423)

**Comment by 2vg** [comment]
@hlaaftana did you say?

> Test failures were from an unrelated C++ framework, this should be opened again

**Comment by hlaaftana** [comment]
Yes, i realized that, but then i checked again and it said mofuw didn't run any tests and that the framework didn't start or whatever

**Comment by 2vg** [comment]
@hlaaftana 
hmm ...
When submitting a pull request to the master branch there was nothing wrong.
However, there seems to be a problem with the Docker branch ... I do not know the cause
